deploy/kustomize/README.md

```diff
@@ -0,0 +1,44 @@
+# Intro
+The KubeVirt Hyperconverged Cluster Operator (HCO) is delivered and deployed on a running OCP/OKD cluster using the kustomize method. 
+
+# Deploy Modes
+HCO can be delivered and deployed in several configurations, corresponding to several pre-built kustomize overlays, which are explained below.
+## Delivery
```

Comment:
nit. In general I find Markdown easier to read when there are empty lines above and below headings.